charts/camunda-platform/values.yaml

```diff
@@ -83,6 +83,16 @@ global:
 
   # Identity configuration to configure identity specifics on global level, which can be accessed by other sub-charts
   identity:
+    # Identity.fullnameOverride can be used to override the full name of the identity resources
+    fullnameOverride: ""
+    # Identity.nameOverride can be used to partly override the name of the identity resources (names will still be prefixed with the release name)
+    nameOverride: ""
+
+    # Identity.service configuration of the identity service on a global level
+    service:
```

Comment:
Since this is a static value, it could be exported the same way as this:
https://github.com/camunda/camunda-platform-helm/blob/1d92a93526e62bc1857e4d7f9cd455cb7c5dd7cb/charts/camunda-platform/Chart.yaml#L29

Reply:
Thanks, I haven't thought about this option :). Using `import-values` would break the configuration though if someone decided to override `identity.service.port`. Probably, that's quite an edge case, but maybe we'd still like to support it. What do you think?

Reply:
Yeah, it's a known limitation.

But we are a step from getting rid of the current structure (umbrella chart with multi-sub charts), and I don't want to introduce anything under `global` (or keep it minimal) to make it easier to migrate later.

So I'd say, as it's not normal flow to customize this value, we can use `import-values` for now.

Reply:
Thanks, I will use it then 👍🏻.